src/components/KDropdownMenu/KDropdownMenu.vue

```diff
@@ -222,9 +222,9 @@ export default defineComponent({
     margin-left: var(--spacing-xs, spacing(xs));
     vertical-align: middle;
     content: "";
-    border-top: 0.325em solid;
-    border-right: 0.325em solid transparent;
-    border-left: 0.325em solid transparent;
+    border-top: 5.2px solid;
+    border-right: 5.2px solid transparent;
+    border-left: 5.2px solid transparent;
```

Comment:
suggestion: if we just round these up to `6px` does it still look ok? 

Same suggestion for all occurrences (e.g. KSelect)

Reply:
Great suggestion, verified after rounding up these still look ok.

**Below screenshots for DropdownMenu and Select compare Prod in Chrome on the left, with LocalDev in Firefox on the right.**

<img width="1721" alt="Screen Shot 2022-12-16 at 10 27 05 AM" src="https://user-images.githubusercontent.com/2568272/208165057-a016fa30-a8f0-430e-96cf-7c1ad345d0f7.png">

<img width="1723" alt="Screen Shot 2022-12-16 at 10 26 27 AM" src="https://user-images.githubusercontent.com/2568272/208165044-a98972d4-301c-4a1e-89f7-79a75d2fca2b.png">